docs/topics/testing/advanced.txt

```diff
@@ -421,19 +421,78 @@ following structure:
 
 .. code-block:: text
 
-    runtests.py
     polls/
         __init__.py
         models.py
         ...
     tests/
         __init__.py
         models.py
-        test_settings.py
+        settings.py
         tests.py
+        urls.py
 
 Let's take a look inside a couple of those files:
 
+.. code-block:: python
+   :caption: ``tests/settings.py``
+
+   from django.urls import include, path
+
+   SECRET_KEY = "fake-key"
+
+   INSTALLED_APPS = [
+       "polls",
+       # adding tests allows us to define models inside the tests package
+       "tests",
+   ]
+
+   DATABASES = {
+       "default": {
+           "ENGINE": "django.db.backends.sqlite3",
+           "NAME": ":memory:",
+       }
+   }
+
+   TEMPLATES = [
+       {
+           "BACKEND": "django.template.backends.django.DjangoTemplates",
+           "APP_DIRS": True,
+       },
+   ]
+
+   ROOT_URLCONF = "tests.urls"
+
+This file contains the :doc:`Django settings </topics/settings>`
+required to run your app's tests.
+
+Since the *tests* package is included in :setting:`INSTALLED_APPS` when
+running your tests, you can define test-only models in its ``models.py``
+file.
+
+
+.. code-block:: python
+   :caption: ``tests/urls.py``
+
+   from django.urls import include, path
+
+   urlpatterns = [
+       path("polls/", include("polls.urls")),
+   ]
+
+If you want to test views, you need a :setting:`ROOT_URLCONF` that includes
+your app.
+
+
+You can then invoke the test runner using
+
+.. code-block:: shell
+
+    python -m django test --settings=tests.settings tests
+
+If you need a more complicated setup, you can create a custom ``runtests.py``
+do manually initialize the test runner.
+
 .. code-block:: python
```

Comment:
This code block needs to replace `test_settings` -> `settings`

Reply:
good catch